libminifi/include/Connection.h

```diff
@@ -180,7 +184,8 @@ class Connection
 	std::atomic<uint64_t> _maxQueueDataSize;
 	//! Flow File Expiration Duration in= MilliSeconds
 	std::atomic<uint64_t> _expiredDuration;
-
+	//! UUID string
+	std::string _uuidStr;
```

Comment:
as i was going through it again I wanted to make clear that this is informational and not a huge deal.